website/sidebars.js

```diff
@@ -170,14 +170,51 @@ export default {
     },
     {
       type: "category",
-      label: "Production Scripts",
+      label: "Ecosystem Scripts",
       className: "category-common-scripts",
       collapsible: false,
       items: [
-        "common-scripts/omnilock",
-        "common-scripts/xudt",
-        "common-scripts/spore-protocol",
-        "common-scripts/spore-dob-0",
+        "common-scripts/introduction",
+        {
+          type: "category",
+          label: "Scripts",
+          className: "scripts",
+          collapsible: true,
+          collapsed: true,
+          items: [
+            "common-scripts/secp256k1_blake160_sighash_all",
```

Comment:
please rename the `common-scripts` to `ecosystem-scripts` too.

Reply:
@joii2020 the please change the url path from `common-scripts` to `ecosystem-scripts` too.